routesrv/redishandler.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 type RedisHandler struct {
-	AddrUpdater func() []byte
+	AddrUpdater []byte
```

Comment:
@MustafaSaber is suspect the `func() ([]byte,error)` respectively here also ?

Reply:
yes, we don't want static addresses which is already supported here https://github.com/zalando/skipper/blob/master/net/redisclient.go#L28